packages/demand-bloc-payment/src/providers/adyenErrors.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpResponseOk, PaymentAuthResponse } from '@karhoo/demand-api'
+
+export const refusalResultCodes = ['Refused', 'Error', 'Cancelled']
+
+export const refusalReasons: Record<string, string> = {
```

Comment:
why do we need to duplicate these errors? Can we import this from `@adyen-web` package ? What if they add a new error, will we need to add it here as well?

Reply:
As far as I understand, adyen npm package doesn't contain API errors definitions.

Reply:
There are no errors description in @adyen-web package, so we need to keep them in our package